### ISSUE TITLE
Blend Image Multiply mode alpha is not working

### ISSUE THREAD

**Issue by iamappmaker** [fabricjs/fabric.js#5316]
## Version
2.4.0

In BlendImage alpha is not working. Both 0 and 1 give same output. 

 var filter1= new fabric.Image.filters.BlendImage({
        image: image,
        mode: 'multiply',
        alpha: 0.5
       });

**Comment by asturur** [comment]
can you make a fiddle? i can fix it but i need an example to verify it

**Comment by asturur** [comment]
i was looking at it, but compared to old filter i have no idea what alpha could do here.... Premultiply the other image pixels to get a softer effect?

**Comment by Tlapi** [comment]
Yes, exactly. It is not working and it could work as a softer effect.

**Comment by iamappmaker** [comment]
BlendImage blends 2 image source and target image. There should be option to control how much source image should blend on target image. In photoshop and we have opacity option to control this.

What alpha does here?
Even in the demo page i can't understand the use of alpha for blend image.

It tried to set opacity of source image but no difference.

For Example See below image. I need to control the opacity of circle or blend level.

![image](https://user-images.githubusercontent.com/43312229/60118116-39fa3480-9799-11e9-9346-af33b9b8e882.png)

**Comment by asturur** [comment]
yes there is a problem with the opacity, the code is missing.

**Comment by iamappmaker** [comment]
One more problem i found. If i apply BlendImage and BlendImage mask to portrait image it crops at the bottom but works correctly for landscape images. 

I am using crop plugin, If i upload any image first it ask for crop image and also i have option to skip. 
If i skip and upload original portrait image, around 20% image in bottom became transparent. If i crop the image and upload it then it works correctly. (there is no relation to crop plugin, it just crop image and send image to fabricjs

**Comment by iamappmaker** [comment]
Thank you. This is such as awesome library helping me to do whatever i want. I have utilized all the features to great extend but there are few major problem i am facing. 
Application become very slow if i use filters, shadows and multiple objects. 
Now i haved set Retina scaling to false (will there be any quality issue ? and is this useful to speed up canvas).

Please suggest me what should i do to increase performance.

**Comment by hmslydia** [comment]
I'm also having this problem. When I use the demo and try to blend two images, it doesn't matter what alpha you pick from the slider, the alpha seems to be 0.5:

http://fabricjs.com/image-filters

**Comment by heavywar** [comment]
with 4.2 not wokring...

**Comment by sops21** [comment]
Seems to not be working in 5.3.0 either
